multihost_test/Bugzillas/test_Regression.py

```diff
@@ -158,3 +158,67 @@ def test_bz487575(self, multihost, create_backup):
         execute_cmd(multihost, "cp -vfr /var/db/nscd_anuj /var/db/nscd")
         execute_cmd(multihost, "rm -rf /home/bz487575dummy*")
         execute_cmd(multihost, "rm -rf /tmp/newusers.txt")
+
+    @pytest.mark.tier1
+    def test_bz218629(self, multihost):
```

Comment:
This test doesn't make sense anymore. The default encryption method for RHEL7, RHEL8 and RHEL9 is SHA512.